x-pack/plugins/security_solution/public/resolver/store/camera/selectors.ts

```diff
@@ -164,7 +164,8 @@ export const scale: (state: CameraState) => (time: number) => Vector2 = createSe
         scalingConstants.maximum
       );
 
-      return (time) => {
+      // memoizing this so the vector returned will be the same object reference if called with the same `time`.
+      return defaultMemoize((time) => {
```

Comment:
any reason to not do this for all of the time based selectors?

Reply:
I didn't do it for `isAnimating` because it returns a primitive and is trivial enough that there should be no perf gain to memoizing.
I didn't do it for `visibleNodesAndEdgeLines` because it just calls two functions which are each already memoized.
I think I did it everywhere else.